tests/expectations/execution/flattened_function_and_inline_matches.out

```diff
@@ -51,7 +51,7 @@ status: accepted
+  "proof": "proof1qyqsqqqqqqqqqqqpqqqqqqqqqqq9xjdh2mkupujkq8w7cznueaxk9jhxzhvs6a5xp0q4cnejllkkrtqzussj034egrrq5l7l3t7asavpqy2a3tmxktqv52uk5kvllxs3fl5tdvz2g9q2e0guuzzgucnfytr92j5sl7v72lkl2qx2s7ep6905pq9v4qjh2a84v8z8c85w9r42czvav4cytns9wafmnfqlw4h6c6cf5tldnzp6stm0stjc4akwgwp3wvqg5vhy9pavhhlamwa4tch7ae9uvxmy22ralrmcqrnyvggdquukqh4jvy2xzma4978axahztszr6h5pz7nxwwlrkdvd47rckqya3wgwen0f5q2wcs6m5ecysw684jhu20ddfx63xpk4999rlq0696d807fczzmw3jeq8t6lej5gycyw9rm23ehfwasup9aw3kvn3ua2slqa7p2kgumxwmrt0awjykzp4sp5uencqpfe5ugvhxjl9xs8qj0k9slfe055zmd5gnpmxyult2ajzdf2cs8amdpy7jfdxz5q2qphwamsc2p48qz357wvkfqujjddelqxu427cryx3kpucrsmq3ptkpcc7eye37pg6nace7x58wcpcaa0dazjvszlukqdlglp7r5ptwr2cdlfrwc2af8dnnqjlgc26658kjpajhu7xrrg8ravq75gax45y7qegap2y95m2h5qjc4teutew3s5y5wr0pjp9t6lp0mj3fuve22rdpex7nu703fxeq9csr2s4x0uhyg38zd790420vwlemcp0uw53e5gz98ddlnyey9kyqgjru7u7f2u8tcn9s0fluk7u43j9fslr2d498urqld86c2q3uttptlggzdyc5v9gm6dpnrkenavkkkh0ac7cnmtt5cjlc7l55yg48lsnvce2h33jvrpuy25esgqc73maede78xgezhaa5gepfjwx24xgss0jxmqh45vac03sd4c6e0572mhmmnlmdq2v5vj69rs6a62qjtslgp8z4lymn3655q9kkzfrd4hlvzscy37e8zyvxlr3v7xfl94eryq7z0gwwz20g06s49zjvqv4aak2dexfsct0kpkutzeyccukhcmuqvypnv2g7c4n6v9vd40crsnqgfnhxdej4mlu2a7h9vrlts734037lpsl75mt8ehdse4mv9mpgngm8u6pme4p2edr6lcw0e33sfxt9yc7jqsqvqqqqqqqqqqq0huegm9wx0e6yqk29cc258046jpj4rajh3v0z6ktxe3qewxgzamkch5rtfgjenz42eqyfmev9pesyqgpnf9nuf2p96a8sut8al7geexgdl2ehad7w8mk7507umh5tzfxy2ufnhheck8zxy9k2tn3ulrusgpqy2zqtu0s2dmv7l9epq5dvgs5dnle9h8eh7fhyq4a2y5kymhhrpqd6tvw285up3ekgwr9p0cyx8du4zht3esrl6a5ze6qvl6tpssjrgwxhzevs2p4j68kgneqqfeqmu2qqqq6y74k7"
```

Comment:
Do we know why the proofs changed? Ditto for below.

Reply:
The curious thing is that the proofs changed, without the compiled bytecode changing. One way to investigate further would be to add an option to fix the randomness for execution tests (if it isn't supported already). Similar to integration tests in snarkVM.